smithy-aws-protocol-tests/model/awsQuery/xml-lists.smithy

```diff
@@ -118,7 +118,29 @@ apply XmlLists @httpResponseTests([
                 }
             ]
         }
-    }
+    },
+    {
```

Comment:
Lots of these tests seem to be testing for different possibilities that clients need to handle, but they don't seem like something a service could actually implement. Can you add a `client-only` tag to client-specific tests to indicate that they are client-specific tests? Like this one: https://github.com/awslabs/smithy/blob/master/smithy-aws-protocol-tests/model/restXml/http-query.smithy#L303